scripts/osfstorage/usage_audit.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+TEMPDIR=`mktemp -d`
+trap "rm -rf $TEMPDIR" EXIT
+
+export HOME=$TEMPDIR
```

Comment:
Why are we resetting `HOME`?

`HOME` is the `sh` variable for the user's home directory - that may be the point of doing this, but it strikes me as a bad idea to change it without ever changing it back.

If we must change it, could we store its value upon entering the script in another variable, then set it back after the script as executed?